docs/modules/network-partitioning/pages/split-brain-recovery.adoc

```diff
@@ -550,4 +551,48 @@ since they are already in the correct in-memory format of the data structure.
 If you return a deserialized value, it might need to be serialized again, which are avoidable costs.
 * Be careful with slow operations in the merge policy (like database accesses),
 since they block your partition threads. Also the `LifeCycleEvent.MERGED` or
-`LifeCycleEvent.MERGE_FAILED` may be thrown too early, if the merge operations take too long to finish.
\ No newline at end of file
+`LifeCycleEvent.MERGE_FAILED` may be thrown too early, if the merge operations take too long to finish.
+
+== Eliminating Unsuccessful Cluster Merges
+
+In some cases, Hazelcast Platform members which initially can form a cluster (join to each other) may not be
+able to recover from the split-brain even though the network failure which caused the split-brain is healed.
+The reason is that the master member of the smaller cluster fails to discover the master member of the bigger cluster, so the split-brain merge process never begins.
+
+Considering a scale-up situation; you add new members after some time since the initial member count in the cluster has become
```

Comment:
```suggestion
Considering the scenario when you are scaling-up a cluster. You need to add new members because the initial member count in the cluster is insufficient.
```